metadata-etl/src/main/resources/jython/ElasticSearchIndex.py

```diff
@@ -145,27 +145,55 @@ def update_dataset(self, last_unixtime=None):
         sql = """
           INSERT IGNORE INTO cfg_search_score_boost
           (id, static_boosting_score)
-          SELECT id, 8 FROM dict_dataset
-          WHERE urn like "kafka:///%" or urn like "oracle:///%" or urn like "espresso:///%";
+          SELECT id, 80 FROM dict_dataset
+          WHERE urn like "kafka:///%"
+             or urn like "oracle:///%"
+             or urn like "espresso:///%"
+          ON DUPLICATE KEY UPDATE
+          static_boosting_score = 80;
+
 
           INSERT IGNORE INTO cfg_search_score_boost
           (id, static_boosting_score)
-          SELECT id, 7 FROM dict_dataset
-          WHERE urn like "hdfs:///%" or urn like "dali:///%";
+          SELECT id, 75 FROM dict_dataset
+          WHERE urn like "dalids:///%"
+          ON DUPLICATE KEY UPDATE
+          static_boosting_score = 75;
+
 
           INSERT IGNORE INTO cfg_search_score_boost
           (id, static_boosting_score)
-          SELECT id, 6 FROM dict_dataset
-          WHERE urn like "hive:///%";
+          SELECT id, 70 FROM dict_dataset
+          WHERE urn like "hdfs:///data/tracking%"
+             or urn like "hdfs:///data/databases%"
+             or urn like "hive:///tracking/%"
+             or urn like "hive:///prod_%"
+          ON DUPLICATE KEY UPDATE
+          static_boosting_score = 70;
+
 
           INSERT IGNORE INTO cfg_search_score_boost
           (id, static_boosting_score)
-          SELECT id, 5 FROM dict_dataset
-          WHERE urn like "%/data/derived%";
+          SELECT id, 65 FROM dict_dataset
+          WHERE urn like "hdfs:///data/external/%"
+             or urn like "hdfs:///data/derived/%"
+             or urn like "hdfs:///data/foundation/%"
+             or urn like "hive:///hirein/%"
+             or urn like "hive:///rightnow/%"
+             or urn like "hive:///lla/%"
+             or urn like "hive:///append_rightnow/%"
+             or urn like "hive:///decipher/%"
+             or urn like "hive:///timeforce/%"
+             or urn like "hive:///jira/%"
+             or urn like "hive:///teleopti/%"
+          ON DUPLICATE KEY UPDATE
+          static_boosting_score = 65;
 
           SELECT * FROM dict_dataset d
           JOIN cfg_search_score_boost s
           WHERE d.id = s.id
+          and urn not like "hive:///dev_foundation_tables%"
```

Comment:
This will eventually replaced by the blacklist provided via parameter in configuration/properties